.gitignore

```diff
@@ -17,3 +17,4 @@ datasets/
 /topicalguide/settings.py
 /working/
 tg_models.png
+/tools/stemmer/
```

Comment:
If you want to ignore files just on your computer, I believe you can add the names of the files/folders to ".git/info/exclude" rather than ".gitignore".  I think we do want to ignore the binaries for everybody though, so maybe we could just add "/tools/stemmer/Snowball" and "/tools/stemmer/stemmer" to .gitignore?